src/drivers/imx/sai.c

```diff
@@ -24,127 +24,107 @@ DECLARE_SOF_UUID("sai", sai_uuid, 0x9302adf5, 0x88be, 0x4234,
 
 DECLARE_TR_CTX(sai_tr, SOF_UUID(sai_uuid), LOG_LEVEL_INFO);
 
-#define REG_TX_DIR 0
-#define REG_RX_DIR 1
+static bool sai_dir_is_synched(struct sai_pdata *sai, bool rx)
```

Comment:
Most of the code looks good, but this stresses me out. We normally use "synced" instead of "synched" as the acronym for "synchronized". That said I won't _mandate_ this.